### PR TITLE
Improve benchmarks

### PR DESCRIPTION
**Pull request by guizmaii**

### CODE REVIEW

File: zio-kafka-bench/src/main/scala/zio/kafka/bench/comparison/KafkaClientBenchmarks.scala [hunk 1]
```diff
@@ -45,15 +59,28 @@ class KafkaClientBenchmarks extends ComparisonBenchmark {
...
-              var count = 0L
+              var count                                            = 0L
+              var first: ConsumerRecord[Array[Byte], Array[Byte]]  = null
+              var lasts: ConsumerRecords[Array[Byte], Array[Byte]] = null
               while (count < numberOfMessages) {
                 val records = consumer.poll(settings.pollTimeout)
                 count += records.count()
+                if (count == 0L) {
+                  first = records.iterator().next()
+                }
```
Comment (erikvanoosten) [inline zio-kafka-bench/src/main/scala/zio/kafka/bench/comparison/KafkaClientBenchmarks.scala:70]: That won't work. When count == 0 records will be empty so next will fail with an exception.

File: .scalafmt.conf
Comment (erikvanoosten) [inline .scalafmt.conf:1]: Please do format changes in a separate PR.

File: .scalafmt.conf [hunk 2]
```diff
@@ -1,21 +1,25 @@
-version=3.7.4
-project.git = true
-maxColumn = 120
-align {
-  preset = more
-  arrowEnumeratorGenerator = true
-}
+version = "3.7.3"
+runner.dialect = Scala213Source3 # https://scalameta.org/scalafmt/docs/configuration.html#scala-2-with--xsource3
+maxColumn = 140
```
Comment (erikvanoosten) [inline .scalafmt.conf:3]: I have a 14" screen, more than 120 is not practical.

File: .scalafmt.conf
Comment (guizmaii) [reply .scalafmt.conf:1]: It's still a draft PR

Comment (erikvanoosten) [reply .scalafmt.conf:1]: Understood, I'll stay away from drafts!
